aten/src/ATen/xpu/XPUContext.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <c10/xpu/XPUFunctions.h>
+
+namespace at::xpu {
+
+// XPU is available if we compiled with XPU.
+inline bool is_available() {
+  return c10::xpu::device_count() > 0;
+}
+
+TORCH_XPU_API DeviceProp* getCurrentDeviceProperties();
+
+TORCH_XPU_API DeviceProp* getDeviceProperties(int device);
+
+TORCH_XPU_API int getGlobalIdxFromDevice(int device);
```

Comment:
The two APIs are global, right? Then, should we use INT to align with other backends? Or should we use unsigned or `DeviceIndex`?

Reply:
Align with other backends. use `int` in public APIs.